src/status_im/contexts/wallet/events.cljs

```diff
@@ -783,17 +783,72 @@
 
 (rf/reg-event-fx
  :wallet/transactions-sent-signal-received
- (fn [{:keys [db]}
+ (fn [_
+      [{sent-transactions :sentTransactions
+        send-details      :sendDetails}]]
+   {:fx [[:dispatch
+          [:wallet/show-transaction-notification
+           {:status           :sent
+            :send-details     send-details
+            :sent-transaction (first sent-transactions)}]]
+         [:dispatch
+          (if (:errorResponse send-details)
+            [:wallet/transaction-failure send-details]
+            [(if (= (:sendType send-details) constants/send-type-swap)
+               :wallet.swap/transaction-success
+               :wallet/transaction-success)
+             sent-transactions])]]}))
+
+(rf/reg-event-fx
+ :wallet/sending-transactions-started-signal-received
```

Comment:
Would be great to move all the router events (e.g. the ones that respond to the router signals) into a separate "router" namespace, cause this one just keeps getting bigger. Could also benefit from namespacing the event keyword e.g. `:wallet.router/transaction-sent-signal-received`, but not necessarily in this PR.